index.html

```diff
@@ -116,10 +118,40 @@ <h2 class="face">Facebook 360</h2>
             <li>css</li>
             <li>javascript</li>
           </ul>
-          <button class="btn">See project</button>
+          <button class="btn" id="myBtn1">See project</button>
         </div>
       </section>
       <!--section 1 of the cards ends-->
+        <div id="myModal" class="modal">
+          <div class="modal-container">
+            <div class="modal-content">
+              <div class="modal-title">
+                <div class="dflex">
+                  <h1 id="modalTitle">Title</h1>
+  
+                  <span class="xp">&times;</span>
+                  </div>
+  
+                <div id="modalSubt" class="modal-nav black"></div>
+                  <img id="modalImg" class="modal-img" src="img/Portfolio1-big.svg" alt="Tonic">
```

Comment:
- [ ] Kindly adjust the padding on the popup modal to match the one in the Figma design. Currently, the height of the modal is so long that, one has to scroll the page down to see the remaining contents of the modal. Please adjust the padding marked in red in the screenshots.
- [ ] It would be nice if you reduce the width of the image a little or you use a smaller or resized version of the images so it helps make the modal frame not exceed the height of the screen which causes vertical scrolling.

**Your Design**
![image](https://user-images.githubusercontent.com/50941074/162087280-dcef343d-3f50-4fd6-a5ec-c80eba0b1e97.png)

**The Figma Design**
![image](https://user-images.githubusercontent.com/50941074/162087403-811dc05d-0b01-4a3d-8794-71d9bc44fec1.png)